sktime/markovprocess/__init__.py

```diff
@@ -4,6 +4,9 @@
 ureg.define('step = []')  # dimensionless unit for unspecified lag time unit.
 Q_ = ureg.Quantity
 
+# TODO: we need to do this for unpickling, but it will overwrite other apps default registry!
```

Comment:
huh well that sucks, serious design flaw in pint imo

Reply:
but thats no better then a temporary fix right? at some point there is going to be a model storing things which aren't dimensionless

Reply:
The problem only arises, when one uses a unit, which isn't defined in the registry used to unpickle it. I think it is very unlikely, that people will be using non-standard time units.

Reply:
Fair enough :)